libs/service-portal/occupational-licenses/src/components/LicenceActionCard.tsx

```diff
@@ -1,14 +1,15 @@
-import { useLocale, useNamespaces } from '@island.is/localization'
+import { useLocale } from '@island.is/localization'
 import { ActionCard } from '@island.is/service-portal/core'
 import { olMessage as ol } from '../lib/messages'
 import { m } from '@island.is/service-portal/core'
+import { OccupationalLicenseStatus } from '@island.is/api/schema'
 
 type LicenseActionCardProps = {
   type?: string
   validFrom?: string | Date
   url?: string
   image?: string
-  isValid: boolean
+  isValid: OccupationalLicenseStatus
```

Comment:
```suggestion
  status: OccupationalLicenseStatus
```

Since this is now an enum `isValid` is not descriptive